src/components/toolbox/buttons/css/baseV2.css

```diff
@@ -27,6 +26,20 @@
   }
 }
 
+@define-mixin buttonDangerV2 {
+  background-color: var(--color-danger) !important;
+  color: var(--color-white) !important;
+
+  &:disabled {
+    opacity: 0.8;
+  }
+
+  &:focus,
+  &:hover {
+    background: #ff6300 !important;
```

Comment:
I think you can use a variable in here

Reply:
Right now it's because this isn't the final color, once we get the correct hover color this should be updated